config/initializers/z_plugin_parsers.rb

```diff
@@ -1,19 +1,21 @@
-# Puppet, the default parser
-Foreman::Plugin.fact_importer_registry.register(:puppet, PuppetFactImporter, true)
-Foreman::Plugin.fact_parser_registry.register(:puppet, PuppetFactParser, true)
+Rails.application.config.to_prepare do
```

Comment:
This looks like a good change. I wonder: with this, is it still needed to have this file named `z_plugin_parsers` or can it now be `plugin_parsers`?